src/map/itemdb.cpp

```diff
@@ -157,15 +148,7 @@ uint64 ItemDatabase::parseBodyNode(const YAML::Node &node) {
 		item->value_buy = buy;
```

Comment:
```suggestion
		item->value_buy = buy;
		item->value_sell = 0;
```
Define `buy` should 'remove' the current `sell`

Reply:
@Atemo only if the entry already exists, correct? Applies to both of your comments.

Reply:
Well yes you're right the values are re-assigned when the entry doesn't exist but it doesn't change the result it's still correct